requirements.txt

```diff
@@ -25,4 +25,7 @@ pip-tools>=6.4.0
 pyparsing==2.4.5
 einops==0.3.2
 protobuf~=3.19.0
-deci-lab-client==2.22.1
\ No newline at end of file
+deci-lab-client==2.22.1
```

Comment:
Double requirement given: deci-lab-client==2.21.1 (from -r requirements.txt (line 31)) (already in deci-lab-client==2.22.1 (from -r requirements.txt (line 28)), name='deci-lab-client')